src/texstudio.cpp

```diff
@@ -7793,7 +7794,7 @@ void Texstudio::previewLatex()
 	}
 	if (!previewc.hasSelection()) {
 		// special handling for cusor in the middle of \[ or \]
-		if (c.previousChar() == '\\' && (c.nextChar() == '[' || c.nextChar() == ']')) {
+		if (c.previousChar() == '\\' and (c.nextChar() == '[' || c.nextChar() == ']')) {
```

Comment:
I would prefer not to have c alternative operators in the source, please revert to '&&'